lib/collections.c

```diff
@@ -87,3 +88,33 @@ void queue_free(Queue* queue)
     }
     free(queue);
 }
+
+// Conch ADT
+typedef struct Conch
+{
+    int64_t conch_value;
+    bool conch_available;
+} Conch;
+
+Conch* conch_init(int64_t initial_conch_value)
+{
+    Conch* conch = malloc(sizeof(Conch));
```

Comment:
According to `man malloc`, the memory isn't necessarily initialized -- so the `conch_available` field could contain abitrary garbage.

Two ideas:
1. assign it to false explicitly
2. supposedly `calloc` is just malloc but it sets the memory to all-zeros, so that could work too